Makefile

```diff
@@ -34,7 +27,7 @@ TEST_SUITE ?= normal
 TEST_TIMEOUT ?= 30m
 test-e2e-no-deploy:
 	# temporary creation of CRD until it lands in openshift/api, openshift/openshift-apiserver, etc.
-	oc apply -f ./deploy/0000_10_projectedresource.crd.yaml
+	oc apply -f vendor/github.com/openshift/api/storage/v1alpha1/0000_10_sharedresource.crd.yaml
```

Comment:
in theory we could just remove this line if we held this for the subsequent openshift/openshift-apiserver PR that would consume the api and client-go PRs you currently noted in you hold

but I'm also fine handling this in a separate PR to avoid adding additional time to this PR merging, waiting for an openshift apiserver PR

Reply:
agreed

Reply:
change looks good

Reply:
Would be good to keep a note of how to apply this change in a local test cluster of sorts. So, we know how to spin up a testing environment using the external dependency which was previously managed by the make target.